src/orquestra/sdk/exceptions.py

```diff
@@ -204,6 +204,12 @@ class WorkflowResultsNotReadyError(NotFoundError):
     """
 
 
+class WorkflowRunIDNotFoundError(NotFoundError):
+    """Raised when we can't recover the ID for this Workflow Run."""
+
```

Comment:
**non-blocking,suggestion**: WDYT about keeping the `wf_run_id` as an attribute of this error? It's a pattern I'd like to adopt whenever adding exceptions because it allows easier pretty-printing by callers like the CLI.

Reply:
Given that its raised when we can't recover the workflow run ID, I'm not sure what utility that would serve?

Reply:
Oooooooh, you're right. We can't assign `self.wf_run_id` because we can't infer it, and it's exactly what's this error is about. Nevermind!